src/k8s-extension/azext_k8s_extension/partner_extensions/OpenServiceMesh.py

```diff
@@ -93,3 +102,63 @@ def Update(self, extension, auto_upgrade_minor_version, release_train, version):
             release_train=release_train,
             version=version
         )
+
+    def Delete(self, client, resource_group_name, cluster_name, name, cluster_type):
+        pass
+
+
+def _validate_tested_distro(cmd, cluster_resource_group_name, cluster_name, extension_version):
+
+    field_unavailable_error = '\"testedDistros\" field unavailable for version {0} of microsoft.openservicemesh, ' \
+        'cannot determine if this Kubernetes distribution has been properly tested'.format(extension_version)
+
+    if version.parse(str(extension_version)) <= version.parse("0.8.3"):
+        logger.warning(field_unavailable_error)

```

Comment:
Why just output the warning log instead of throwing an exception here?

Reply:
This is a partner ask for us, for versions below this, they want to alert people that it is unstable but not break current users

Reply:
Got it, thanks~